package.json

```diff
@@ -46,5 +46,8 @@
     "twilio": "^4.20.0",
     "twilio-video": "^2.29.0",
     "vue": "^3.5.13"
+  },
+  "optionalDependencies": {
+    "electron-installer-redhat": "latest"
```

Comment:
This is needed for electron windows to avoid pesky permission errors at `clean` stage